Modules/System/Azure File Services API/src/Helper/AFSFormatHelper.Codeunit.al

```diff
@@ -0,0 +1,77 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+codeunit 8955 "AFS Format Helper"
+{
+    Access = Internal;
+    InherentEntitlements = X;
+    InherentPermissions = X;
+
+    [NonDebuggable]
+    procedure AppendToUri(var Uri: Text; ParameterIdentifier: Text; ParameterValue: Text)
```

Comment:
This should use UriBuilder. The current implementation does not encode the values, it may also be possible for non-admin users to mess up with the call if they have access to set just one of these parameters.